plugins/org/src/components/Cards/OwnershipCard/OwnershipCard.tsx

```diff
@@ -71,6 +71,7 @@ export const OwnershipCard = (props: {
   relationsType?: EntityRelationAggregation;
   relationAggregation?: EntityRelationAggregation;
   entityLimit?: number;
+  title?: string;
```

Comment:
I recommend using the same type declared for the `InfoCard` title 🙂 :
https://github.com/backstage/backstage/blob/master/packages/core-components/src/layout/InfoCard/InfoCard.tsx#L137